### PR TITLE
Updated test options in 'CONTRIBUTING.md'.

### PR DESCRIPTION
**Pull request by DanielRosenwasser**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -75,14 +80,15 @@ jake runtests tests=2dArrays debug=true
...
+To add a new test case, simply place a `.ts` file in `tests\cases\compiler` containing code that exemplifies the bugfix or change you are making.
 
 These files support metadata tags in the format  `// @metaDataName: value`. The supported names and values are:
 
-* `comments`, `sourcemap`, `noimplicitany`, `declaration`: true or false (corresponds to the compiler command-line options of the same name)
-* `target`: ES3 or ES5 (same as compiler)
-* `out`, outDir: path (same as compiler)
-* `module`: local, commonjs, or amd (local corresponds to not passing any compiler --module flag)
+* `comments`, `sourcemap`, `noimplicitany`, `declaration`: `true` or `false` (corresponds to the compiler command-line options of the same name)
+* `target`: `ES3`, `ES5`, `ES6`, `ES2015`, or `ES7` (same as compiler)
```
Comment (mhegazy) [inline CONTRIBUTING.md]: ES7 is not a valid target.additionally, we should stop using numbers, it should be ES2016

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -75,14 +80,15 @@ jake runtests tests=2dArrays debug=true
...
+
+To add a new test case, simply place a `.ts` file in `tests\cases\compiler` containing code that exemplifies the bugfix or change you are making.
 
 These files support metadata tags in the format  `// @metaDataName: value`. The supported names and values are:
 
-* `comments`, `sourcemap`, `noimplicitany`, `declaration`: true or false (corresponds to the compiler command-line options of the same name)
-* `target`: ES3 or ES5 (same as compiler)
-* `out`, outDir: path (same as compiler)
-* `module`: local, commonjs, or amd (local corresponds to not passing any compiler --module flag)
+* `comments`, `sourcemap`, `noimplicitany`, `declaration`: `true` or `false` (corresponds to the compiler command-line options of the same name)
```
Comment (mhegazy) [inline CONTRIBUTING.md]: we have changed all of these now.. so any configuration that works in a tsconfig should work in the tests. 
the only additions worth mentioning is fileName, see the additions here:
https://github.com/Microsoft/TypeScript/blob/master/src/harness/harness.ts#L977-L983

Comment (DanielRosenwasser) [inline CONTRIBUTING.md]: So what exactly should I write here?

Comment (mhegazy) [inline CONTRIBUTING.md]: i think that:
all compiler options can be passed to the test harness by prefixing them with `@` e.g. `@removeComments: true` or `@outFile: a\b.js`

then leave the entry that already exits about fileName

**Comment by mhegazy** [comment]
two comments above, other than that LGTM

**Pull request merged**